src/main/scala/scalaz/stream/writer.scala

```diff
@@ -0,0 +1,20 @@
+package  scalaz.stream
+
+import scalaz.\/._
+
+
+object writer {
+
+  /** Promote a `Process` to a `Writer` that writes nothing. */
+  def lift[F[_], A](p: Process[F, A]): Writer[F, Nothing, A] =
```

Comment:
@pchlupacek  Sure, I'm adding `liftO` and `liftW` right now, so paste away!

@pchiusano I might have had them backwards before. This is what's on my branch now:

```scala
  /** Promote a `Process` to a `Writer` that writes nothing. */
  def liftW[F[_], O](p: Process[F, O]): Writer[F, Nothing, O] =
    p.map(right)

  /** Promote a `Process` to a `Writer` that outputs nothing. */
  def liftO[F[_], W](p: Process[F, W]): Writer[F, W, Nothing] =
    p.map(left)
```